apps/fluent-tester/metro.config.js

```diff
@@ -7,41 +7,55 @@
 
 const path = require('path');
 const { exclusionList, makeMetroConfig, resolveUniqueModule } = require('@rnx-kit/metro-config');
+const MetroSymlinksResolver = require('@rnx-kit/metro-resolver-symlinks');
 
-const [reactIs, reactIsExcludePattern] = resolveUniqueModule('react-is');
+// ensure regex paths are merged, normalized, use forward slashes and end with a /
+function pathForRegex(...parts) {
+  let result = path.normalize(path.join(...parts));
+  if (!result.endsWith(path.sep)) {
+    result += path.sep;
+  }
+  return result.replace(/[/\\]+/g, '/');
+}
 
-const blockList = exclusionList([
-  /node_modules\/.*\/node_modules\/react-native\/.*/,
+const excludeMixins = [];
+const extraNodeModules = {};
+function ensureUniqueModule(moduleName, excludeList, nodeModules) {
+  const [nmEntry, excludePattern] = resolveUniqueModule(moduleName);
+  excludeMixins.push(excludePattern);
+  extraNodeModules[moduleName] = nmEntry;
+}
+
+// build up the added excludes and extraNodeModules
+['react-is', 'invariant', '@babel/runtime', 'base64-js'].forEach((moduleName) => ensureUniqueModule(moduleName));
 
+const blockList = exclusionList([
   // This stops "react-native run-windows" from causing the metro server to
-  // crash if its already running
-  new RegExp(`${path.join(__dirname, 'windows').replace(/[/\\]+/g, '/')}.*`),
+  // crash if its already running. This should also cover /.*\/.vs\/.*/, as .vs folders go next to the .sln file
+  new RegExp(`${pathForRegex(__dirname, 'windows')}.*`),
 
   // Workaround for `EPERM: operation not permitted, lstat '~\midl-MIDLRT-cl.read.1.tlog'`
-  /.*\.tlog/,
-
-  // Prevent Metro from watching temporary files generated by Visual Studio
-  // otherwise it may crash when they are removed when closing a project.
-  /.*\/.vs\/.*/,
+  /.*\.tlog$/,
```

Comment:
This should be removable. It's included in default config: https://github.com/microsoft/rnx-kit/blob/42267b575a8c24b23c4c85a7c11bc47e142aece9/packages/metro-config/src/index.js#L270-L271